lib/corebulk.go

```diff
@@ -79,9 +79,9 @@ type BulkIndexer struct {
 	// Buffer for Max number of time before forcing flush
 	BufferDelayMax time.Duration
 	// Max buffer size in bytes before flushing to elasticsearch
-	BulkMaxBuffer int // 1048576
+	BulkMaxBufferCB func() int // 1048576
```

Comment:
Maybe remove the default comment

Reply:
Gah, the default comment is also wrong (the default is 16kB, not 1MB as that suggests). Will remove.